hw03-annotation/src/main/java/TestClass.java

```diff
@@ -0,0 +1,31 @@
+import annotation.After;
+import annotation.Before;
+import annotation.Test;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class TestClass {
+    private static final Logger logger = LoggerFactory.getLogger(TestClass.class);
+
+    @Before
+    void setUpTest() {
+        logger.info("Before");
+    }
```

Comment:
Если исключение будет в Before-методе, то посмотрите на ваше поведение и поведение JUnit.

```suggestion
    void setUpTest() {
        logger.info("Before");
        throw new RuntimeException("fail test");
    }
```